docs/rollout/introduce-tilt.md

```diff
@@ -1,63 +1,80 @@
 * See Tilt perform the update and then see the change live
+* Optionally: suggest a file and line at which to introduce an error; see the app crash and see Tilt report the error.
 
-An example workflow makes it easy for a user to map Tilt onto their existing mental model of the dev workflow.
-
-## Advanced Features
+An example workflow makes it easy for a user to map Tilt onto their existing mental model of the dev workflow. It's also a great way to drive home the advantages of Tilt over your old tool. For instance, if resetting the database used to require an elaborate terminal incantation and can now be done by simply triggering a [`local_resource`](../local_resource) from the Web UI, make that part of your example workflow. (If you had a group of [Focus Users](../rollout/focus) alpha-testing your Tilt setup, the features they were most excited about will generally be good ones to mention here.)
 
-As you configured Tilt you may have added convenient features in the Tiltfile. E.g. making it easy to recreate the database with a [`local_resource`](../local_resource) or use config values to choose what servers to set up as editable. Tell users about the bells and whistles so they can enjoy them too!
+### Known Issues
 
-## Known Issues
+As you first build out Tilt support, there will probably be ways that Tilt is worse than the legacy tool---or even just different. Telling users about these regressions before they run into them builds trust. Something as simple as a list of links to issues in your issue tracker helps manage users' expectations. (As noted above, many perceived "regressions" stem from misconceptions about how Tilt works, or are necessary trade-offs for big benefits down the road: explaining this context can help your users feel less annoyed.)
 
-At the beginning there are probably ways that Tilt is worse or even just different than the legacy dev environment. Telling users about these differences before they experience them builds trust. Soemthing as simple as a list of links to issues in your issue tracker helps manage users' expectations.
+### Next Steps
 
-## Next Steps
+Tell users how to give feedback and get support (by joining a Slack channel, subscribing to an email list, etc.). If you're looking for volunteers to help improve the Tilt experience, mention that here.
 
-Tell users how to give feedback and get support. Mabye they should join a Slack channel or subscribe to an email list. If you're looking for volunteers to help improve Tilt, mention that here.
+Your goal in this section should be to catch underwhelmed users before they fall out of the funnel---it's better to get their feedback now than to hunt them down later as [inactive users](../rollout/prioritize-inactive). The easier (or more mandatory) you make it for AppDevs to give you feedback, the greater the chance an underwhelmed AppDev will come give you useful information (rather than giving up and walking away).
```

Comment:
```suggestion
Your goal in this section should be to catch underwhelmed users before they give up entirely---it's better to get their feedback now than to hunt them down later as [inactive users](../rollout/prioritize-inactive). The easier (or more mandatory) you make it for AppDevs to give you feedback, the greater the chance an underwhelmed AppDev will come give you useful information (rather than giving up and walking away).
```

It seems better to drop "funnel" to 1) not burden readers with thinking about that and 2) sound less growth-hacky

Reply:
good call